yt_dlp/extractor/youtube.py

```diff
@@ -1917,35 +2201,66 @@ def get_text(x):
 
         ytm_streaming_data = {}
         if is_music_url:
-            # we are forcing to use parse_json because 141 only appeared in get_video_info.
-            # el, c, cver, cplayer field required for 141(aac 256kbps) codec
-            # maybe paramter of youtube music player?
-            ytm_player_response = self._parse_json(try_get(compat_parse_qs(
-                self._download_webpage(
-                    base_url + 'get_video_info', video_id,
-                    'Fetching youtube music info webpage',
-                    'unable to download youtube music info webpage', query={
-                        **self._get_video_info_params(video_id),
-                        'el': 'detailpage',
-                        'c': 'WEB_REMIX',
-                        'cver': '0.1',
-                        'cplayer': 'UNIPLAYER',
-                    }, fatal=False) or ''),
-                lambda x: x['player_response'][0],
-                compat_str) or '{}', video_id, fatal=False)
-            ytm_streaming_data = ytm_player_response.get('streamingData') or {}
-
+            ytm_webpage = None
+            sts = self._extract_signature_timestamp(video_id, player_url, ytcfg, fatal=False)
+            if sts and 'youtube-limit-requests' not in compat_opts:  # TODO: better compat opt name
```

Comment:
I'm thinking there should be a way to toggle downloading these extra configs (and use the hard-coded ones), just to save on number of requests/help with rate limiting.